plugins/plugin-babel/worker.js

```diff
@@ -2,8 +2,8 @@ const workerpool = require('workerpool');
 const babel = require('@babel/core');
 
 async function transformFileAsync(path, options) {
-  const {code, map} = await babel.transformFileAsync(path, options);
-  return JSON.stringify({code, map});
+  const {contents, code, map} = await babel.transformFileAsync(path, options);
```

Comment:
why this change? what's the difference between `contents` and `code` in babel? Lets use only one